dataland-inbound-proxy/scripts/obtain-letsencrypt-certs.sh

```diff
@@ -10,6 +10,7 @@ done
 if [ ! -d "/etc/letsencrypt/live" ] && [ ! -f "/certs/custom/privkey.pem" ]; then
   echo "Requesting LetsEncrypt certificates"
   mkdir -p /var/www/certbot && certbot certonly --non-interactive --webroot --webroot-path /var/www/certbot --agree-tos --preferred-challenges http $PROXY_LETSENCRYPT_ARGS
+  /scripts/reload-certificates.sh
```

Comment:
wie besprochen bitte noch ändern:
In `configure-certificate-link.sh`:
* 
```  
rm /certs/dataland
ln -s /certs/dummy /certs/dataland
```
ändern auf 
```  
ln -sf /certs/dummy /certs/dataland
```
* die Logik aus `generat-dummy-certs` ins Dockerfile auslagern - das reicht wenn's @Compile-Time passiert - muss nicht bei @StartTime passieren.
* Entsprechend in `configure-certificate-link.sh` den dritten Case entfernen.

Reply:
ok